src/components/FileSelector/FileSelector.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Dropzone from 'react-dropzone';
+import {
+  FormElement,
+  FormElementControl,
+  FormElementError,
+} from '../Form';
+import { Button } from '../Button';
+
+const FileSelector = ({
+  accept,
+  buttonText,
+  id,
+  error,
+  fatalError,
+  label,
+  multiple,
+  onDragEnter,
+  onDragLeave,
+  onDrop,
+  selectorText,
+  selectorType,
+}) => {
+  const errorStr = fatalError || error;
```

Comment:
What use cases we have for non-fatal error?

Reply:
Why not just expose `isDisabled` prop? IMHO it will be more universal. We may have situations where we want to disable FileSelector without any errors

Reply:
Non-fatal error: An uploaded file was rejected but we can accept other uploads
Fatal error: The file selector is not usable due to missing permissions etc.

Salesforce does not expose `disabling` without an error string according to the docs, I don't think that we should break that expectation